### DISCUSSION TITLE
Add manual examples to CI

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#47044] [1. Feature Evaluation]
### Is your feature request related to a problem? Please describe.
We are working on step by step examples for developers. The same extensions can and should be used for automated ci tests.

### Describe the solution you'd like
For each PR, download the installation files from https://github.com/joomla/manual-examples and install them.
Find a way to make this as extendable as possible, maybe only a configuration file is needed which is adding one line in the cms ci which allows to download and run specific tests from this extension.

### Additional context

@muhme agreed to implement such system.

**Comment by muhme** [comment]
May 22nd created PR https://github.com/joomla/manual-examples/pull/14 (waiting for merge) with first version running automated Cypress tests for all existing modul-tutorial samples

**Comment by muhme** [comment]
with [manual-examples/pull/28](https://github.com/joomla/manual-examples/pull/28) (change installation to ZIP-file) the implementation is completed for me and from my side this issue can be closed

**Comment by HLeithner** [comment]
Thanks, now we need to integrate it into the cms ci tests I asked @Hackwar if he can help here or maybe @laoneo has time for it
